Add line for releases to README

connormcdermid opened the pull request:
readme change

connormcdermid committed:
Created Sounds folder, added music track

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

connormcdermid committed:
Added 'Death and Axes' music as potential boss theme

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

connormcdermid committed:
guys, I found it, I found the chase music

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

anmolbaidwan committed:
Added Sounds script

FindObjectOfType<AudioManager>().Play("Goofy Ahh Death"); is all thats needed to play a sound from any script, added sound for player shooting, enemy death, and player death.

anmolbaidwan committed:
Added volume float to function

now volume of all sounds can be altered, added a parameter. values can be from 0.0 to 1.0 for volume. also added comet death sound

anmolbaidwan committed:
Added Goofy sounds toggle

Currently there is a toggle button in the pause menu to switch between "Goofy" sounds and normal sounds. Sounds currently implemented are Enemy and Player Death, and Weapon Fire for Player (both goofy and regular versions) + goofy comet strike.

connormcdermid committed:
Add line for releases to README

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

Q-ACT committed:
Cobus Takes Damage

Q-ACT committed:
Added Factorial Bullet to Cobus

Q-ACT committed:
Cobus Spawn Aligns with Health bar fill

Q-ACT committed:
Merge branch 'testing' into sounds

anmolbaidwan committed:
Merge pull request #59 from wehaventdecided/sounds

Merge Sounds into Testing

LukaKaranovic reviewed (approved):
LGTM
Happy Coding!
-Luka

Code-Persona reviewed (approved):
LGTM WEEEEEEEEEEE

connormcdermid committed:
included GameControl

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

connormcdermid committed:
added PlaySound to a namespace so it works properly

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

connormcdermid committed:
Disabled goofy mode, renamed sounds

Goofy mode sounds mismatched with regular sounds. Goofy toggle has been disabled.

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

connormcdermid committed:
missed a file whoops

Signed-off-by: Connor McDermid <mcdermidconnor@outlook.com>

anmolbaidwan reviewed (approved):
giggity
